### PR TITLE
performance: reminders alerts

### PR DESCRIPTION
**Pull request by gymad**
## Description

performance improvement for alert notifications

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [x] My code follows the code style of this project found [here](https://suitecrm.com/wiki/index.php/Coding_Standards).
- [ ] My change requires a change to the documentation.
- [ ] I have read the [**How to Contribute**](https://suitecrm.com/wiki/index.php/Contributing_to_SuiteCRM) guidelines.

### CODE REVIEW

**Comment by pgorod** [comment]
@gymad you might want to edit the title to include "Fix #5313" which is the Issue number. Thanks

**Comment by daniel-samson** [comment]
I think it would be a good idea to right an automated test - which creates a large number of alerts and then checks the performance.

File: modules/Reminders/vardefs.php [hunk 1]
```diff
@@ -102,6 +102,16 @@
             'massupdate' => false,
             'studio' => false,
         ),
+        'date_willexecute' => array(
+            'name' => 'date_willexecute',
+            'vname' => 'LBL_DATE_WILLEXECUTE',
```
Comment (horus68) [inline modules/Reminders/vardefs.php]: This string should be also created in a language file? LBL_DATE_WILLEXECUTE

**Comment by pgorod** [comment]
Dillon should this really be closed without merging? I see there is another PR for 7.8.x, but doesn't this belong also in 7.9.x?

**Pull request closed without merging**